src/components/ListUniqueValues.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div class="list-unique-values">
+    <div>FILTER VALUES</div>
+    <input
+      class="list-unique-values__search-box"
+      placeholder="Search 50 items"
+      @input="search($event.target.value)"
+    />
+    <div class="list-unique-values__select-clear-all">
+      <span class="list-unique-values__select-all" @click="selectAll">Select all</span> &nbsp;
+      <span class="list-unique-values__clear-all" @click="clearAll">Clear all</span>
+    </div>
+    <div
+      class="list-unique-values__checkbox-container"
+      v-for="value in searchedValues"
+      :key="value.value"
+    >
+      <CheckboxWidget
+        :label="`${value.value} (${value.nbOcc})`"
+        :value="isChecked[value.value]"
+        @input="toggleCheck(value.value)"
+      />
+    </div>
+    <div class="list-unique-values__load-more" v-if="loaded">
+      <div>List maybe incomplete</div>
+      <div @click="loadMore" class="list-unique-values__load-more-button">Load more</div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+// This type will be imported from @/lib/dataset/helpers.ts
+type ColumnValueStat = {
+  value: any;
+  nbOcc: number;
+};
+
+import CheckboxWidget from './stepforms/widgets/Checkbox.vue';
+@Component({
+  name: 'filter-value',
+  components: { CheckboxWidget },
+})
+export default class ActionMenu extends Vue {
+  @Prop({
+    type: Array,
+    required: true,
+  })
+  values!: ColumnValueStat[];
+
+  @Prop({
+    type: Boolean,
+    required: true,
+  })
+  loaded!: boolean;
+
+  searchedValues: ColumnValueStat[] = this.values;
+
+  isChecked: Record<string, boolean> = Object.fromEntries(this.values.map(e => [e.value, true]));
+
+  selectAll() {
+    this.isChecked = Object.fromEntries(this.values.map(e => [e.value, true]));
+    this.$emit('input', Object.keys(this.isChecked));
+  }
+
+  clearAll() {
+    this.isChecked = Object.fromEntries(this.values.map(e => [e.value, false]));
+    this.$emit('input', []);
+  }
+
+  toggleCheck(value: string) {
+    this.isChecked[value] = !this.isChecked[value];
+    this.$emit(
+      'input',
+      Object.keys(this.isChecked).filter(k => {
+        return this.isChecked[k];
+      }),
+    );
+  }
+
+  search(search: string) {
+    this.searchedValues = this.values.filter(k => {
+      return this.searchFunction(k.value.toLowerCase(), search.toLowerCase());
+    });
+  }
+
+  // TODO: enhance searchFunction
+  searchFunction(value: string, search: string) {
+    return value.startsWith(search);
+  }
+
+  loadMore() {
+    // trigger store action to load more
+  }
+}
+</script>
+<style lang="scss" scoped>
+.list-unique-values {
+  max-height: 300px;
+  overflow: auto;
+  background-color: #fafafa; //$grey
```

Comment:
I feel we're gonna have a lot of different grays if we don't start using variables here ^^
May we add the color to `src/styles/_variables.scss` for further reuse?